cli/command/container/cp.go

```diff
@@ -272,13 +327,27 @@ func copyToContainer(ctx context.Context, dockerCli command.Cli, copyConfig cpCo
 
 		resolvedDstPath = dstDir
 		content = preparedArchive
+		content = &cpWithProgress{ReadCloser: content, toContainer: true, containerName: copyConfig.container, copyPath: dstInfo.Path, quiet: copyConfig.quiet, writer: dockerCli.Out(), total: &copiedSize}
 	}
 
 	options := types.CopyToContainerOptions{
 		AllowOverwriteDirWithFile: false,
 		CopyUIDGID:                copyConfig.copyUIDGID,
 	}
-	return client.CopyToContainer(ctx, copyConfig.container, resolvedDstPath, content, options)
+
+	if !copyConfig.quiet {
+		fmt.Fprint(dockerCli.Out(), "Preparing to copy...\n")
+		fmt.Fprint(dockerCli.Out(), "\033[s")
```

Comment:
Hm.. I'm bad at these; what's this sequence doing?

I have a feeling that a `cursor-up` is missing; here's the output after a copy;

```console
$ docker cp --quiet=false foo:/usr/share/nginx bla
Preparing to copy...
Copying from container - 4.608kB
Successfully copied 4.608kB to /go/src/github.com/docker/cli/bla
```

I think the `Preparing to copy...` needs to be overwritten by `Copying from container ...`

Perhaps for readability, we could use the https://github.com/morikuni/aec package, which we already use in other places, e.g.

```suggestion
		fmt.Fprint(dockerCli.Out(), "\033[s")
		fmt.Fprint(dockerCli.Out(), aec.Up(1))
```

Reply:
Uhm, I'll check this with your on your PR.